### PR TITLE
Update success message for addMedCon and delMedCon

### LINKED ISSUE #130: Rename medCon add and delete success message
Change success message such that it returns
the uric of patient instead of their name to prevent
risk of confusion from duplicate names for doctors

### PR DESCRIPTION
**Pull request by TechRaj**
Fixes #130

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddMedConCommand.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public class AddMedConCommand extends Command {
             + PREFIX_NRIC + "T0123456F "
             + "c/Diabetes c/Hypertension";
 
-    public static final String MESSAGE_ADD_MEDCON_SUCCESS = "Added medical condition: %1$s to Person: %2$s";
+    public static final String MESSAGE_ADD_MEDCON_SUCCESS = "Added medical condition: %1$s to Nric: %2$s";
```
Comment (yong-zhuo) [inline src/main/java/seedu/address/logic/commands/AddMedConCommand.java:32]: Good fix

**Review by yong-zhuo — APPROVED** [review]
Overall LGTM to merge

File: src/main/java/seedu/address/logic/commands/AddMedConCommand.java (hunk 1 above)
Comment (hweinian) [inline src/main/java/seedu/address/logic/commands/AddMedConCommand.java:32]: good fix

File: src/main/java/seedu/address/logic/commands/DelMedConCommand.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ private String generateSuccessMessage(Person personToEdit) {
 
         String resultMedCon = '[' + medConsString.toString() + ']';
 
-        return String.format(MESSAGE_DELETE_MEDCON_SUCCESS, resultMedCon, personToEdit.getName().fullName);
+        return String.format(MESSAGE_DELETE_MEDCON_SUCCESS, resultMedCon, personToEdit.getNric().value);
```
Comment (hweinian) [inline src/main/java/seedu/address/logic/commands/DelMedConCommand.java:95]: lgtm

**Pull request merged**